src/classes/CW_MarcomBriefViewControllerTest.cls

```diff
@@ -0,0 +1,166 @@
+/**
+* File Name   :    CW_MarcomBriefViewController.cls
+* Description :    B-109868: Add Related objects to the MarCom Brief View 
+*
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 07/06/2014 Cristina P.	Intial
+*
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.